django_netjsongraph/api/generics.py

```diff
@@ -38,7 +54,7 @@ class BaseReceiveTopologyView(APIView):
     parser_classes = (TextParser,)
 
     def post(self, request, pk, format=None):
-        topology = get_object_or_404(self.model, pk, strategy='receive')
+        topology = get_object_or_404(self.topology_model, pk, strategy='receive')
```

Comment:
as a general rule, never change lines which are working fine and do not need to be changed, unless you have a very valid reason to do so